Makefile

```diff
@@ -169,8 +169,12 @@ push:
 
 .PHONY: sign
 sign:
+	docker pull $(DOCKER_IMAGE_NAME):$(IMMUTABLE_DOCKER_TAG)
+	docker pull $(DOCKER_IMAGE_NAME):$(MUTABLE_DOCKER_TAG)
 	docker trust sign $(DOCKER_IMAGE_NAME):$(IMMUTABLE_DOCKER_TAG)
 	docker trust sign $(DOCKER_IMAGE_NAME):$(MUTABLE_DOCKER_TAG)
+	docker trust inspect --pretty $(DOCKER_IMAGE_NAME):$(IMMUTABLE_DOCKER_TAG)
+	docker trust inspect --pretty $(DOCKER_IMAGE_NAME):$(MUTABLE_DOCKER_TAG)
```

Comment:
I figure this is a good addition so we can see proof that signing worked right in the build logs instead of having to manually verify it post-release.